resources/sass/_base.scss

```diff
@@ -51,18 +51,16 @@ $caret-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5
     }
   }
 
-  &::selection,
-  &::-moz-selection,
-  &::-webkit-selection {
-    background: $border-color-hover;
-    color: $text-color;
+  &::selection {
```

Comment:
Sure, made this:

https://github.com/kint-php/kint/pull/430

I assumed it would be fine to integrate it here at first, in order to reduce clutter and avoid having to open multiple PRs.